cms/tests/test_apphooks.py

```diff
@@ -32,7 +32,7 @@
 APP_MODULE = "cms.test_utils.project.sampleapp.cms_apps"
 
 
-class ApphooksTestCase(CMSTestCase):
+class ApphooksTestCase(ClearURLs, CMSTestCase):
```

Comment:
What's the root cause of the issue though? Is django-cms doing something "weird" or is it a bug in Django itself?